core/src/consensus/tendermint/stake/action_data.rs

```diff
@@ -31,6 +33,12 @@ fn get_account_key(address: &Address) -> H256 {
     blake256(rlp.drain())
 }
 
+fn get_stakeholder_addresses_key() -> H256 {
```

Comment:
How about using lazy_static instead of calculating it every time?

Reply:
You can see how to use lazy_static in https://github.com/CodeChain-io/codechain/blob/de496666005edf8744b1108b30b6975600bea232/util/logger/src/lib.rs#L49-L51